components/Benefit/Benefit.scss

```diff
@@ -0,0 +1,175 @@
+.benefit {
+  padding: 2rem 0;
+
+  &__wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  &__card {
+    position: relative;
+    margin-bottom: 1rem;
+    border-radius: 2.3rem;
+    padding: 1.5rem;
+    width: 100%;
+    min-height: 16rem;
+    box-sizing: border-box;
```

Comment:
Let's move this to upper level

```
html, body {
  box-sizing: border-box;
}
```

Probably you'll need to recalculate some of the widths